.npmrc

```diff
@@ -1,4 +1,3 @@
-public-hoist-pattern[] = *slidev*
 public-hoist-pattern[] = *types*
```

Comment:
[suggestion](https://conventionalcomments.org/#labels): The entirety of this file can now be deleted since pnpm works well with SvelteKit. These hacks were only needed for sli.dev